pom.xml

```diff
@@ -277,6 +251,15 @@
                             <rpt>SCRIPT_STYLE</rpt>
                         </mapping>
                     </configuration>
+                    <dependencies>
+                        <dependency>
+                            <!-- includes common copyright headers for sources -->
+                            <groupId>org.kaazing</groupId>
+                            <artifactId>community.license</artifactId>
+                            <type>jar</type>
+                            <version>${project.version}</version>
```

Comment:
The release process of community updates version numbers listed in any pom file (as seen by the [version in master](https://github.com/kaazing/community/blob/master/community.main/pom.xml) to 2.20).  It does NOT replace variable references (presumably because you might actually want the variable).  The release process of components/repositories that consume community don't replace references to constants, but do replace variables.  I'm not sure which release process you are referencing, but I think that answers your question, right?